docs/how-to/run-services-reliably.md

```diff
@@ -0,0 +1,74 @@
+# How to run services reliably
+
+Microservice architectures offer flexibility, but they can introduce reliability challenges such as network interruptions, resource exhaustion, problems with dependent services, cascading failures, and deployment issues. Health checks can address these issues by monitoring resource usage, checking the availability of dependencies, catching problems of new deployments, and preventing downtime by redirecting traffic away from failing services.
+
+To help you manage services more reliably, Pebble provides a comprehensive health check feature.
+
+## Use health checks of the HTTP type
+
+A health check of the HTTP type issues HTTP `GET` requests to the health check URL at a user-specified interval.
+
+The health check is considered successful if the check returns an HTTP 200 response. After getting a certain number of failures in a row, the health check is considered "down" (or unhealthy).
```

Comment:
This is actually 2xx (see also [this minor issue I just fixed](https://github.com/canonical/pebble/pull/565)).
```suggestion
The health check is considered successful if the URL returns any HTTP 2xx response. After getting a certain number of failures in a row, the health check is considered "down" (or unhealthy).
```